plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/nessie/IcebergNessieCatalogConfig.java

```diff
@@ -67,4 +74,45 @@ public IcebergNessieCatalogConfig setDefaultWarehouseDir(String defaultWarehouse
         this.defaultWarehouseDir = defaultWarehouseDir;
         return this;
     }
+
+    @MinDuration("1ms")
+    public Duration getReadTimeout()
+    {
+        return readTimeout;
+    }
+
+    @Config("iceberg.nessie-catalog.read-timeout")
+    @ConfigDescription("The read timeout for the client.")
+    public IcebergNessieCatalogConfig setReadTimeout(Duration readTimeout)
+    {
+        this.readTimeout = readTimeout;
+        return this;
+    }
+
+    @MinDuration("1ms")
+    public Duration getConnectionTimeout()
+    {
+        return connectionTimeout;
+    }
+
+    @Config("iceberg.nessie-catalog.connection-timeout")
+    @ConfigDescription("The connection timeout for the client.")
+    public IcebergNessieCatalogConfig setConnectionTimeout(Duration connectionTimeout)
+    {
+        this.connectionTimeout = connectionTimeout;
+        return this;
+    }
+
+    public boolean isCompressionDisabled()
+    {
+        return disableCompression;
+    }
+
+    @Config("iceberg.nessie-catalog.disable-compression")
```

Comment:
`iceberg.nessie-catalog.disable-compression` is a bit too generic - consider another property name which identifies better what this setting is about.

consider also flip it  to `enable-compression`

Reply:
https://github.com/projectnessie/nessie/blob/8e70d3a49720e3688b3feef533bced607faaec92/api/client/src/main/java/org/projectnessie/client/NessieConfigConstants.java#L235-L239

Reply:
IDE gave a warning that the usage is always the inverse. So, the property name can be disable instead of enable. Hence, I inversed it.

Reply:
Let's make this usable for humans, not for IDEs :) 

Think positive.